### ISSUE TITLE
Independent t-test

### ISSUE THREAD

**Issue by kaz94** [rasbt/mlxtend#807]
Comparing the same algorithm but on different data sets, and deciding whether the difference in a chosen metric is statistically significant or just due to randomness could be useful.

**Comment by rasbt** [comment]
It might be useful to at 2-sample independent t-test in general, but I think using either (a) a 5x2cv paired t test or  McNemar test is better for classifier comparisons

- http://rasbt.github.io/mlxtend/user_guide/evaluate/paired_ttest_5x2cv/
- http://rasbt.github.io/mlxtend/user_guide/evaluate/mcnemar/

**Comment by kaz94** [comment]
Yes, I suggested it since I needed to use the independent version to
compare the same classifier but on different data sets :)


niedz., 14 mar 2021, 16:57 użytkownik Sebastian Raschka <
***@***.***> napisał:
